### ISSUE TITLE
Request: Scully Icon

### ISSUE THREAD

**Issue by atao60** [simple-icons/simple-icons#4355]
**Name:** Scully
**Website:** https://scully.io/
**Alexa rank:** [363,249 (2020/12/14)](https://www.alexa.com/siteinfo/scully.io)

**Official resources for icon and color:**
https://github.com/scullyio/scully/tree/main/assets/logos

**Comment by fbernhart** [comment]
**Source:** There is a SVG file without the wordmark provided in their [GitHub repo](https://github.com/scullyio/scully/blob/d4df05bf2811ef13f08e4759dfc829322e52ac86/assets/logos/SVG/scullyio-icon.svg)
**Color:** [Their website](https://scully.io/) seems to use `#1BAC4E` for headings, buttons etc. So I would go for this color, as the logo uses a gradient of different shades of green.

**Comment by atao60** [comment]
What about using:
- **source**: the png white icon provided with file [scullyio-icon-white.png](https://raw.githubusercontent.com/scullyio/scully/d4df05bf2811ef13f08e4759dfc829322e52ac86/assets/logos/PNG/White/scullyio-icon-white.png)
- **color**: the background color *Scully Green* <code>#3AA860<span style="background-color: #3AA860; height: 0.8em; width: 0.8em;"></span></code> as specified in file [scullyio-logo-source.ai](https://github.com/scullyio/scully/raw/d4df05bf2811ef13f08e4759dfc829322e52ac86/assets/logos/scullyio-logo-source.ai).

**Comment by fbernhart** [comment]
@atao60 It's always better to use a SVG version (if available) directly than to convert a PNG file to an SVG file manually.

Concerning the color: Thanks for looking into the scullyio-logo-source.ai file, I haven't done it yet. I agree that `#3AA860` would be a better fit in this case. 👍🏼

**Comment by atao60** [comment]
About SVG: it's also available with [scullyio-icon-white.svg](https://raw.githubusercontent.com/scullyio/scully/d4df05bf2811ef13f08e4759dfc829322e52ac86/assets/logos/SVG/White/scullyio-icon-white.svg)

**Comment by atao60** [comment]
My workaround in the meantime:
```
svn export https://github.com/scullyio/scully/trunk/assets/logos/SVG/White/scullyio-icon-white.svg src/assets/img/scullyio-icon-white.svg

IMG_URL="$(npx url-cli -p "$(echo "data:image/svg+xml;base64,$(openssl base64 -A -in src/assets/img/scullyio-icon-white.svg)")")"

sed -i 's|logo=scully|logo='$IMG_URL'|' README.md

```
Just what I need...

**Comment by PeterShaggyNoble** [comment]
Would you like to have a read through [our contributing guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md) and give this a try yourself, @atao60?

**Comment by atao60** [comment]
Would you allow attribute 'aria-label' on tag 'svg' in the svg file?

**Comment by fbernhart** [comment]
@atao60 No, unfortunately not. See #1191 and https://github.com/simple-icons/simple-icons/issues/1135#issuecomment-454674687 for details.

The SVG should follow this format:

```
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Scully icon</title><path d="..."/></svg>
```

More details can be found in our [contribution guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md).

**Comment by atao60** [comment]
For my own edification:

```bash
### POC logo creation only with tools on command line (and without Inkscape)
#   
#    prerequisites: svn, xmllint , xmlstarlet, svg-path-bbox, svgo, ifne + sponge (moreutils under Ubuntu), sed & bc !

git clone --filter=tree:0 git@github.com:atao60/simple-icons.git simple-icons && cd $_

git checkout develop
# Already on 'develop'
# Your branch is up to date with 'origin/develop'.

git checkout -b issue4355
# Switched to a new branch 'issue4355'

code .

### Folder Icon is already in .gitignore
mkdir -pv Icon && cd $_
# mkdir: created directory 'Icon'

svn export https://github.com/scullyio/scully/trunk/assets/logos/SVG/White/scullyio-icon-white.svg .
# A    scullyio-icon-white.svg
# Export complete.

cp scullyio-icon-white.svg scully.svg

cat scully.svg | sed -E -n 's/^.*?(xmlns="[^"]*").*$/\1\n/p'
# xmlns="http://www.w3.org/2000/svg"

sed -i 's/xmlns="[^"]*"//' scully.svg

xmllint --xpath "string(//svg/@viewBox)" scully.svg
# 0 0 340 340

xmlstarlet ed --update "//svg/title" --value "Scully icon" scully.svg | ifne sponge scully.svg

xmlstarlet ed --update "//svg/@viewBox" --value "0 0 24 24" scully.svg | ifne sponge scully.svg

xmlstarlet ed --insert "//svg" --type "attr" --name "role" --value "img" scully.svg | ifne sponge scully.svg

### See https://github.com/simple-icons/simple-icons/issues/4355#issuecomment-758319246
# xmlstarlet ed --insert "//svg" --type "attr" --name "aria-label" --value "Scully" scully.svg | ifne sponge scully.svg

xmlstarlet ed --delete "//svg/path/@fill" scully.svg | ifne sponge scully.svg

npx svg-path-bbox "$(xmllint --xpath "string(//svg/path/@d)" scully.svg)"
# npx: installed 2 in 1.64s
# 19.99034753895546 19.981401820976423 319.99362852819064 320.00697302091953

echo "319.99362852819064 - 19.99034753895546" | bc
# 300.00328098923518

echo "320.00697302091953 - 19.981401820976423" | bc
# 300.025571199943107

### Take the same value for x and y, i.e. 300
#   The final image must fit in a box of 24 x 24, so:
echo "scale=5; 24 / 300" | bc
# .08000

### Any transformations added here will be merged to the path by 'svgo'
#   Dixit http://www.w3.org/TR/SVG/coords.html#TransformAttribute
#          If a list of transforms is provided, then the net effect is as if each 
#          transform had been specified separately in the order provided.
#   but from right to left!
#   Here to avoid the left upper corner to be recalculated with 'scale',
#   proceed first with translation to set up its left upper corner at coordinates (0, 0)
xmlstarlet ed --insert "//svg/path" --type "attr" --name "transform" \
  --value " scale(0.08) translate(-19.99034753895546,-19.981401820976423)" \
  scully.svg | ifne sponge scully.svg
  
xmlstarlet ed --insert "//svg" --type "attr" --name "xmlns" --value "http://www.w3.org/2000/svg" scully.svg | ifne sponge scully.svg
  
(cd .. && npm run svgo -- ./Icon/scully.svg)

cat scully.svg
# <svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Scully icon</title><path d="M22.377 1.193a1.72 1.72 0 0 0-1.4-1.064A22.6 22.6 0 0 0 .329 9.25a1.72 1.72 0 0 0-.176 1.72 22.616 22.616 0 0 0 7.24 8.88 22.864 22.864 0 0 0 10.36 4.136 1.688 1.688 0 0 0 1.584-.656 22.768 22.768 0 0 0 4.432-10.488 22.64 22.64 0 0 0-1.392-11.648zm-11.768 3.44a8.936 8.936 0 0 1 3.24-.504c1.696.056 3.672.456 5.584 2.952a.864.864 0 0 1-.888 1.376.864.864 0 0 1-.496-.32c-1.2-1.568-2.336-2.216-4.256-2.272a7.36 7.36 0 0 0-2.544.384.872.872 0 1 1-.64-1.616zm6.904 13.48a8.72 8.72 0 0 1-3.232.504 8.176 8.176 0 0 1-1.56-.2c-1.448-.328-3.096-1.128-4.472-2.936a.872.872 0 1 1 1.384-1.048c1.2 1.568 2.776 2.392 4.696 2.448a7.208 7.208 0 0 0 2.552-.384.868.868 0 0 1 .632 1.616zm2.12-2.768a.92.92 0 0 1-1 .608l-.176-.04a.928.928 0 0 1-.568-1.184c.288-.824.016-1.2-.072-1.32-.712-1.008-3.144-1.096-4.152-1.088-2.472.016-4.128-.592-4.928-1.808s-.488-2.512-.208-3.136a.928.928 0 0 1 1.696.744c-.008.024-.32.808.064 1.384s1.616.976 3.36.96c2.872-.016 4.776.608 5.68 1.864.6.84.704 1.88.304 3.016z"/></svg>

git branch
#   develop
# * issue4355

cp scully.svg ../icons

sed -i '/^\s*"title"\s*:\s*"Seagate"/ i\
            "title": "Scully",\
            "hex": "3AA860",\
            "source": "https://github.com/scullyio/scully/tree/d4df05bf2811ef13f08e4759dfc829322e52ac86/assets/logos"\
        },\
        {
' ../_data/simple-icons.json

npm i

npm run lint

### Push to create a pull request

cd ..

git add .

git commit -m "Add Scully (#4355)"

git push origin issue4355

```

**Comment by adamrusted** [comment]
Wow, their Alexa rank has really [dropped off a cliff](https://www.alexa.com/siteinfo/scully.io) since this issue was first created. Any ideas why, @simple-icons/maintainers? Currently the Alexa Rank and GitHub stars would unfortunately mean this is out of scope for us.

**Comment by atao60** [comment]
I don't know how to answer...
- for me https://github.com/simple-icons/simple-icons/issues/4355#issuecomment-756682428 was a clear green light to do the job. It's hard to understand such a stop and go policy :disappointed:, even if the status of one of your criteria has dramatically changed.
- about Alexa: what about the period of the cliff, i.e. from 18 to 24 of December (500k threshold the 23)? Christmas eve maybe :wink:
- about Alexa (bis): the 500k threshold has been crossed the 23 and now the rank seems quite stabilized just above it
- about GitHub stars: even if it's a young project, just one year, its stars' curve follows strong growth, see https://seladb.github.io/StarTrack-js/#/preload?r=scullyio,scully. There is no reason that will end now: Scully is the first Angular SSG (and it seems the only one ?) available off the shelf. Furthermore it's a good framework :innocent:

Best wishes for 2021.

**Comment by ericcornelissen** [comment]
@atao60 apologies for that. The 500k Alexa rank threshold is not a strict one, neither are the GitHub stars and Scully _may_ still be included if those "requirements" are not met. In any case, if what you say is true, and _Scully_ continues to see strong growth, the SVG can always be included whenever that happens, so you work will in #4355 has not been for nothing :slightly_smiling_face:

I do hope you understand that we need some cutoff point for brands to include. People will always say their brand will become popular, but unfortunately saying that doesn't make it true. We as maintainer just need some time to investigate Scully and see if what you say is true. Even if we consider it out of scope for now, we would be happy to include Scully once it is definitely popular enough :+1:

**Comment by adamrusted** [comment]
> * for me [#4355 (comment)](https://github.com/simple-icons/simple-icons/issues/4355#issuecomment-756682428) was a clear green light to do the job. It's hard to understand such a stop and go policy 😞, even if the status of one of your criteria has dramatically changed.

Apologies, I meant to just flag it to the other maintainers, not to diminish the work you've put into this PR. The icon itself looks great, and as @ericcornelissen mentions, once the icon is back within scope then we can of course add it.

> * about Alexa: what about the period of the cliff, i.e. from 18 to 24 of December (500k threshold the 23)? Christmas eve maybe 😉

It may well be that people haven't been looking at Scully over the holidays - so I think it's only sensible for us to keep an eye on it for a few more weeks and see what happens. I'll mark this issue and PR as `pending` for now, subject to us seeing an improvement in the rankings.

Thanks for your work on this one @atao60 - hopefully we can merge your PR in the near future! 🤞🏻🎉
